visidata/loaders/postgres.py

```diff
@@ -52,6 +52,10 @@ def openurl_postgres(url, filetype=None):
     return PgTablesSheet(dbname+"_tables", sql=SQL(conn))
 
 
+def openurl_postgresql(url, filetype):
```

Comment:
Thanks for putting this together, @zormit !  Let's do `openurl_postgresql=openurl_postgres` and save a line of code :)

Reply:
haha of course